daemon/k8s_watcher.go

```diff
@@ -986,9 +987,23 @@ func (d *Daemon) nodesAddFn(obj interface{}) {
 		return
 	}
 	ni := node.Identity{Name: k8sNode.Name}
-
 	n := k8s.ParseNode(k8sNode)
-	node.UpdateNode(ni, n)
+
+	routeTypes := node.BPFRoute
+
+	// Add IPv6 routing only in non encap. With encap we do it with bpf tunnel
```

Comment:
Does `encap` refer to http://docs.cilium.io/en/latest/concepts/#direct-native-routing-mode , or something else?

Reply:
Yes, it is that encap